opae.spec.fedora

```diff
@@ -16,7 +16,7 @@ Requires:  systemd
 Requires:  tbb
 Requires:  hwloc
 Requires:  spdlog
-Requires:  libedit
+Requires:  edit
```

Comment:
We don't want this.  Please omit this change.

Reply:
reverted change back libedit